docs/guides/get-qpu-information.ipynb

```diff
@@ -246,6 +246,41 @@
     "For a full list of attributes, see the [`IBMBackend` API documentation](/docs/api/qiskit-ibm-runtime/ibm-backend)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "6a055938-0a1d-46ef-ae9c-60a7960d6d73",
+   "metadata": {},
+   "source": [
+    "## Native gates and operations\n",
+    "\n",
+    "Each [processor family](/docs/guides/processor-types) has a native gate set. By default, the QPUs in each family only support running the gates and operations in the native gate set. Thus, every gate in the circuit must be translated (by the transpiler) to the elements of this set.\n",
+    "\n",
+    "You can view the native gates and operations for a QPU either [with Qiskit](#native-gates-with-qiskit), or on the IBM Quantum&reg; Platform [Compute resources page](/docs/guides/qpu-information#native-gates-on-platform).\n",
+    "\n",
+    "<Admonition type=\"note\" title=\"Native gates and basis gates\">\n",
+    "The terms native gates and basis gates are often used interchangeably. The native gates are those that calibrations have been done for on the QPU, so the QPU can understand and execute them without further instruction. However, you can pass a subset of the native gates as the basis gates to the transpiler. For information about changing the basis gates, see the [Represent quantum computers](/docs/guides/represent-quantum-computers#basis-gates) topic.\n",
+    "</Admonition>\n",
```

Comment:
After reviewing the `QPU information` section, I think we should just delete this. This paragraph was more useful when you could add custom gates via pulse gates... but that's no longer supported. It's now confusing to have these as separate terms since IQP uses `basis gates` to mean `native gates`.